src/utilities/currency/index.test.ts

```diff
@@ -0,0 +1,44 @@
+import { format, parse } from '.';
+
+describe('format', () => {
+  test('should format Currency into BRL', () => {
+    expect(format(1)).toBe('1,00');
```

Comment:
Missing `0.01` and `0.10`

Reply:
Well noted, I'll cover these cases too 👍